doc/parameters.txt

```diff
@@ -1,6 +1,19 @@
 Commandline Parameters
 ======================
 
+In what follows, "Startup" parameters are parameters that are passed to the
+Linux kernel, which will pass those it did not consume to the script starting
```

Comment:
The preinit script gets the kernel command-line from `/proc/cmdline` which is the untouched kernel command-line. i.e. it includes parameters used by the kernel directly . These would get passed to the installer and ignored.

Reply:
I've removed the concept of consuming from the text. Is it accurate now?